[I 16:55:51.261 LabApp] JupyterLab extension loaded from /Users/jacobyeung/anaconda3/lib/python3.8/site-packages/jupyterlab
[I 16:55:51.262 LabApp] JupyterLab application directory is /Users/jacobyeung/anaconda3/share/jupyter/lab
[I 16:55:51.266 LabApp] Serving notebooks from local directory: /Users/jacobyeung/Berkeley/Spring 2021/CS 182/CS-182-Project
[I 16:55:51.266 LabApp] The Jupyter Notebook is running at:
[I 16:55:51.266 LabApp] http://localhost:8890/?token=d842084f935286e6d11ec68fd23a49f6269e4d099c66cbc3
[I 16:55:51.266 LabApp]  or http://127.0.0.1:8890/?token=d842084f935286e6d11ec68fd23a49f6269e4d099c66cbc3
[I 16:55:51.266 LabApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 16:55:51.287 LabApp] 
    
    To access the notebook, open this file in a browser:
        file:///Users/jacobyeung/Library/Jupyter/runtime/nbserver-12192-open.html
    Or copy and paste one of these URLs:
        http://localhost:8890/?token=d842084f935286e6d11ec68fd23a49f6269e4d099c66cbc3
     or http://127.0.0.1:8890/?token=d842084f935286e6d11ec68fd23a49f6269e4d099c66cbc3
[I 16:55:55.085 LabApp] Build is up to date
[W 16:55:55.595 LabApp] 404 GET /api/contents/StyleTransfer.ipynb?content=0&1616975755575 (127.0.0.1): No such file or directory: StyleTransfer.ipynb
[W 16:55:55.595 LabApp] No such file or directory: StyleTransfer.ipynb
[W 16:55:55.596 LabApp] 404 GET /api/contents/StyleTransfer.ipynb?content=0&1616975755575 (127.0.0.1) 3.75ms referer=http://localhost:8890/lab
[W 16:55:55.599 LabApp] 404 GET /api/contents/LSTM_Captioning.ipynb?content=0&1616975755575 (127.0.0.1): No such file or directory: LSTM_Captioning.ipynb
[W 16:55:55.599 LabApp] No such file or directory: LSTM_Captioning.ipynb
[W 16:55:55.599 LabApp] 404 GET /api/contents/LSTM_Captioning.ipynb?content=0&1616975755575 (127.0.0.1) 3.06ms referer=http://localhost:8890/lab
[W 16:55:55.599 LabApp] 404 GET /api/contents/lstm_pytorch.ipynb?content=0&1616975755575 (127.0.0.1): No such file or directory: lstm_pytorch.ipynb
[W 16:55:55.599 LabApp] No such file or directory: lstm_pytorch.ipynb
[W 16:55:55.600 LabApp] 404 GET /api/contents/lstm_pytorch.ipynb?content=0&1616975755575 (127.0.0.1) 3.15ms referer=http://localhost:8890/lab
[W 16:55:55.603 LabApp] 404 GET /api/contents/deeplearning/style_transfer.py?content=0&1616975755576 (127.0.0.1): No such file or directory: deeplearning/style_transfer.py
[W 16:55:55.604 LabApp] No such file or directory: deeplearning/style_transfer.py
[W 16:55:55.604 LabApp] 404 GET /api/contents/deeplearning/style_transfer.py?content=0&1616975755576 (127.0.0.1) 5.75ms referer=http://localhost:8890/lab
[W 16:55:55.605 LabApp] 404 GET /api/contents/deeplearning/rnn_layers.py?content=0&1616975755576 (127.0.0.1): No such file or directory: deeplearning/rnn_layers.py
[W 16:55:55.605 LabApp] No such file or directory: deeplearning/rnn_layers.py
[W 16:55:55.606 LabApp] 404 GET /api/contents/deeplearning/rnn_layers.py?content=0&1616975755576 (127.0.0.1) 1.15ms referer=http://localhost:8890/lab
[W 16:55:55.607 LabApp] 404 GET /api/contents/deeplearning/captioning_solver.py?content=0&1616975755576 (127.0.0.1): No such file or directory: deeplearning/captioning_solver.py
[W 16:55:55.607 LabApp] No such file or directory: deeplearning/captioning_solver.py
[W 16:55:55.607 LabApp] 404 GET /api/contents/deeplearning/captioning_solver.py?content=0&1616975755576 (127.0.0.1) 1.47ms referer=http://localhost:8890/lab
TermSocket.open: 1
[I 16:55:55.667 LabApp] New terminal with specified name: 1
TermSocket.open: Opened 1
[I 17:00:38.226 LabApp] Creating new notebook in /
[I 17:00:38.493 LabApp] Kernel started: cf22b87e-98dd-4d54-af5a-ab5d751a8cfe
[I 17:00:39.445 LabApp] Starting buffering for cf22b87e-98dd-4d54-af5a-ab5d751a8cfe:447b240f-8080-465f-be98-d0600afa8b25
[I 17:02:38.708 LabApp] Saving file at /Untitled.ipynb
[I 17:06:38.752 LabApp] Saving file at /Untitled.ipynb
[I 17:08:38.787 LabApp] Saving file at /Untitled.ipynb
[I 17:10:38.818 LabApp] Saving file at /Untitled.ipynb
[I 17:12:38.856 LabApp] Saving file at /Untitled.ipynb
[I 17:13:02.503 LabApp] Saving file at /Untitled.ipynb
[I 23:06:34.258 LabApp] interrupted
Serving notebooks from local directory: /Users/jacobyeung/Berkeley/Spring 2021/CS 182/CS-182-Project
1 active kernel
The Jupyter Notebook is running at:
http://localhost:8890/?token=d842084f935286e6d11ec68fd23a49f6269e4d099c66cbc3
 or http://127.0.0.1:8890/?token=d842084f935286e6d11ec68fd23a49f6269e4d099c66cbc3
Shutdown this notebook server (y/[n])? [C 23:06:36.292 LabApp] received signal 2, stopping
[I 23:06:36.298 LabApp] Shutting down 1 kernel
[I 23:06:36.813 LabApp] Kernel shutdown: cf22b87e-98dd-4d54-af5a-ab5d751a8cfe
Traceback (most recent call last):
  File "/Users/jacobyeung/anaconda3/lib/python3.8/site-packages/traitlets/config/loader.py", line 254, in __getitem__
    return dict.__getitem__(self, key)
KeyError: 'nbserver_extensions'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jacobyeung/anaconda3/bin/jupyter-lab", line 10, in <module>
    sys.exit(main())
  File "/Users/jacobyeung/anaconda3/lib/python3.8/site-packages/jupyter_core/application.py", line 270, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/Users/jacobyeung/anaconda3/lib/python3.8/site-packages/traitlets/config/application.py", line 663, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-7>", line 2, in initialize
  File "/Users/jacobyeung/anaconda3/lib/python3.8/site-packages/traitlets/config/application.py", line 87, in catch_config_error
    return method(app, *args, **kwargs)
  File "/Users/jacobyeung/anaconda3/lib/python3.8/site-packages/notebook/notebookapp.py", line 1767, in initialize
    self.init_server_extension_config()
  File "/Users/jacobyeung/anaconda3/lib/python3.8/site-packages/notebook/notebookapp.py", line 1654, in init_server_extension_config
    self.config.NotebookApp.nbserver_extensions.update({modulename: enabled})
  File "/Users/jacobyeung/anaconda3/lib/python3.8/site-packages/traitlets/config/loader.py", line 279, in __getattr__
    return self.__getitem__(key)
  File "/Users/jacobyeung/anaconda3/lib/python3.8/site-packages/traitlets/config/loader.py", line 262, in __getitem__
    v = LazyConfigValue()
  File "/Users/jacobyeung/anaconda3/lib/python3.8/site-packages/traitlets/traitlets.py", line 958, in __new__
    inst.setup_instance(*args, **kwargs)
  File "/Users/jacobyeung/anaconda3/lib/python3.8/site-packages/traitlets/traitlets.py", line 982, in setup_instance
    def setup_instance(self, *args, **kwargs):
KeyboardInterrupt
[I 13:33:53.869 LabApp] JupyterLab extension loaded from /Users/jacobyeung/anaconda3/lib/python3.8/site-packages/jupyterlab
[I 13:33:53.889 LabApp] JupyterLab application directory is /Users/jacobyeung/anaconda3/share/jupyter/lab
[I 13:33:53.893 LabApp] Serving notebooks from local directory: /Users/jacobyeung/Berkeley/Spring 2021/CS 182/CS-182-Project
[I 13:33:53.893 LabApp] The Jupyter Notebook is running at:
[I 13:33:53.893 LabApp] http://localhost:8890/?token=ea7d546bea01b41f9cf75e5e822fdcc87e739eda5f816246
[I 13:33:53.893 LabApp]  or http://127.0.0.1:8890/?token=ea7d546bea01b41f9cf75e5e822fdcc87e739eda5f816246
[I 13:33:53.893 LabApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 13:33:53.950 LabApp] 
    
    To access the notebook, open this file in a browser:
        file:///Users/jacobyeung/Library/Jupyter/runtime/nbserver-92884-open.html
    Or copy and paste one of these URLs:
        http://localhost:8890/?token=ea7d546bea01b41f9cf75e5e822fdcc87e739eda5f816246
     or http://127.0.0.1:8890/?token=ea7d546bea01b41f9cf75e5e822fdcc87e739eda5f816246
[I 13:33:58.675 LabApp] Build is up to date
[W 13:33:59.519 LabApp] Notebook timer.ipynb is not trusted
[W 13:33:59.575 LabApp] Notebook validation.ipynb is not trusted
[I 13:34:00.174 LabApp] Kernel started: 589d02ec-fa8c-466a-9a4c-1d706251e037
[I 13:34:00.187 LabApp] Kernel started: 993b95e5-dffd-432e-bed8-491bad038c20
[I 13:34:01.026 LabApp] Starting buffering for 993b95e5-dffd-432e-bed8-491bad038c20:759af332-66fb-4c69-bcd8-636efbd4925f
[I 13:34:24.476 LabApp] Creating new notebook in /
[I 13:34:24.705 LabApp] Kernel started: d01f93a2-7419-42ee-9e91-1c5736ff69f6
[I 13:36:00.070 LabApp] Saving file at /validation.ipynb
[W 13:36:00.071 LabApp] Notebook validation.ipynb is not trusted
[I 13:36:24.663 LabApp] Saving file at /Untitled.ipynb
[I 13:38:24.702 LabApp] Saving file at /val_creator.ipynb
[I 13:40:24.731 LabApp] Saving file at /val_creator.ipynb
[I 13:42:24.851 LabApp] Saving file at /val_creator.ipynb
[I 13:44:24.882 LabApp] Saving file at /val_creator.ipynb
[I 13:46:24.913 LabApp] Saving file at /val_creator.ipynb
[I 13:46:48.798 LabApp] Saving file at /val_creator.ipynb
[I 13:47:00.968 LabApp] interrupted
Serving notebooks from local directory: /Users/jacobyeung/Berkeley/Spring 2021/CS 182/CS-182-Project
3 active kernels
The Jupyter Notebook is running at:
http://localhost:8890/?token=ea7d546bea01b41f9cf75e5e822fdcc87e739eda5f816246
 or http://127.0.0.1:8890/?token=ea7d546bea01b41f9cf75e5e822fdcc87e739eda5f816246
Shutdown this notebook server (y/[n])? [C 13:47:03.358 LabApp] received signal 2, stopping
[I 13:47:03.361 LabApp] Shutting down 3 kernels
[C 13:47:03.571 LabApp] received signal 2, stopping
[I 13:47:03.673 LabApp] Kernel shutdown: 589d02ec-fa8c-466a-9a4c-1d706251e037
[I 13:47:03.675 LabApp] Kernel shutdown: 993b95e5-dffd-432e-bed8-491bad038c20
[I 13:47:03.676 LabApp] Kernel shutdown: d01f93a2-7419-42ee-9e91-1c5736ff69f6
[I 20:18:15.456 LabApp] JupyterLab extension loaded from /Users/jacobyeung/anaconda3/lib/python3.8/site-packages/jupyterlab
[I 20:18:15.457 LabApp] JupyterLab application directory is /Users/jacobyeung/anaconda3/share/jupyter/lab
[I 20:18:15.461 LabApp] Serving notebooks from local directory: /Users/jacobyeung/Berkeley/Spring 2021/CS 182/CS-182-Project
[I 20:18:15.461 LabApp] The Jupyter Notebook is running at:
[I 20:18:15.461 LabApp] http://localhost:8890/?token=189a7011530bed82ae7be79d928ce928b0d54185d0781855
[I 20:18:15.461 LabApp]  or http://127.0.0.1:8890/?token=189a7011530bed82ae7be79d928ce928b0d54185d0781855
[I 20:18:15.461 LabApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 20:18:15.481 LabApp] 
    
    To access the notebook, open this file in a browser:
        file:///Users/jacobyeung/Library/Jupyter/runtime/nbserver-86135-open.html
    Or copy and paste one of these URLs:
        http://localhost:8890/?token=189a7011530bed82ae7be79d928ce928b0d54185d0781855
     or http://127.0.0.1:8890/?token=189a7011530bed82ae7be79d928ce928b0d54185d0781855
[I 20:18:18.978 LabApp] Build is up to date
[W 20:18:19.387 LabApp] 404 GET /api/contents/Imitation_Learning.ipynb?content=0&1619320699381 (127.0.0.1): No such file or directory: Imitation_Learning.ipynb
[W 20:18:19.387 LabApp] No such file or directory: Imitation_Learning.ipynb
[W 20:18:19.387 LabApp] 404 GET /api/contents/Imitation_Learning.ipynb?content=0&1619320699381 (127.0.0.1) 1.54ms referer=http://localhost:8890/lab
[I 20:18:29.323 LabApp] interrupted
Serving notebooks from local directory: /Users/jacobyeung/Berkeley/Spring 2021/CS 182/CS-182-Project
0 active kernels
The Jupyter Notebook is running at:
http://localhost:8890/?token=189a7011530bed82ae7be79d928ce928b0d54185d0781855
 or http://127.0.0.1:8890/?token=189a7011530bed82ae7be79d928ce928b0d54185d0781855
Shutdown this notebook server (y/[n])? [C 20:18:31.217 LabApp] received signal 2, stopping
[I 20:18:31.217 LabApp] Shutting down 0 kernels
[I 2021-04-24 20:19:14.078 ServerApp] jupyterlab | extension was successfully linked.
[I 2021-04-24 20:19:14.509 ServerApp] nbclassic | extension was successfully linked.
[I 2021-04-24 20:19:14.560 LabApp] JupyterLab extension loaded from /Users/jacobyeung/anaconda3/envs/ml/lib/python3.7/site-packages/jupyterlab
[I 2021-04-24 20:19:14.560 LabApp] JupyterLab application directory is /Users/jacobyeung/anaconda3/envs/ml/share/jupyter/lab
[I 2021-04-24 20:19:14.566 ServerApp] jupyterlab | extension was successfully loaded.
[I 2021-04-24 20:19:14.573 ServerApp] nbclassic | extension was successfully loaded.
[I 2021-04-24 20:19:14.573 ServerApp] Serving notebooks from local directory: /Users/jacobyeung/Berkeley/Spring 2021/CS 182/CS-182-Project
[I 2021-04-24 20:19:14.574 ServerApp] Jupyter Server 1.2.3 is running at:
[I 2021-04-24 20:19:14.574 ServerApp] http://localhost:8890/lab?token=390f84fe10f632729c09f62e66a3b2622821680e801884a2
[I 2021-04-24 20:19:14.574 ServerApp]  or http://127.0.0.1:8890/lab?token=390f84fe10f632729c09f62e66a3b2622821680e801884a2
[I 2021-04-24 20:19:14.574 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2021-04-24 20:19:14.591 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///Users/jacobyeung/Library/Jupyter/runtime/jpserver-86169-open.html
    Or copy and paste one of these URLs:
        http://localhost:8890/lab?token=390f84fe10f632729c09f62e66a3b2622821680e801884a2
     or http://127.0.0.1:8890/lab?token=390f84fe10f632729c09f62e66a3b2622821680e801884a2
[I 2021-04-24 20:19:19.615 LabApp] Build is up to date
[I 2021-04-24 20:24:15.192 ServerApp] Creating new notebook in 
[I 2021-04-24 20:24:15.529 ServerApp] Kernel started: f25c90ee-1d1d-4782-a60d-d13571a1f580
[I 2021-04-24 20:26:15.435 ServerApp] Saving file at /prune.ipynb
[I 2021-04-24 20:27:53.786 ServerApp] Creating new file in 
[I 2021-04-24 20:28:16.315 ServerApp] Saving file at /prune.ipynb
[I 2021-04-24 20:35:51.023 ServerApp] Saving file at /model_creator.py
[I 2021-04-24 20:36:17.350 ServerApp] Saving file at /prune.ipynb
[I 2021-04-24 20:36:57.706 ServerApp] Saving file at /model_creator.py
[I 2021-04-24 20:37:08.148 ServerApp] Saving file at /model_creator.py
[I 2021-04-24 20:37:30.150 ServerApp] Saving file at /model_creator.py
[I 2021-04-24 20:37:37.924 ServerApp] Saving file at /model_creator.py
[I 2021-04-24 20:38:17.384 ServerApp] Saving file at /prune.ipynb
[I 2021-04-24 20:42:41.343 ServerApp] Saving file at /model_creator.py
[I 2021-04-24 20:43:40.636 ServerApp] Kernel restarted: f25c90ee-1d1d-4782-a60d-d13571a1f580
[I 2021-04-24 20:44:17.438 ServerApp] Saving file at /prune.ipynb
[I 2021-04-24 20:46:17.474 ServerApp] Saving file at /prune.ipynb
[I 2021-04-27 03:38:43.437 ServerApp] Starting buffering for f25c90ee-1d1d-4782-a60d-d13571a1f580:20961afa-826f-4626-8df5-7ef408805849
[I 16:21:18.099 LabApp] The port 8890 is already in use, trying another port.
[I 16:21:18.385 LabApp] JupyterLab extension loaded from /Users/jacobyeung/anaconda3/lib/python3.8/site-packages/jupyterlab
[I 16:21:18.386 LabApp] JupyterLab application directory is /Users/jacobyeung/anaconda3/share/jupyter/lab
[I 16:21:18.389 LabApp] Serving notebooks from local directory: /Users/jacobyeung/Berkeley/Spring 2021/CS 182/CS-182-Project
[I 16:21:18.389 LabApp] The Jupyter Notebook is running at:
[I 16:21:18.390 LabApp] http://localhost:8891/?token=c4a7922a4e79cd1b3fe89771477cf0d924ef322c9ed45fad
[I 16:21:18.390 LabApp]  or http://127.0.0.1:8891/?token=c4a7922a4e79cd1b3fe89771477cf0d924ef322c9ed45fad
[I 16:21:18.390 LabApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 16:21:18.408 LabApp] 
    
    To access the notebook, open this file in a browser:
        file:///Users/jacobyeung/Library/Jupyter/runtime/nbserver-36223-open.html
    Or copy and paste one of these URLs:
        http://localhost:8891/?token=c4a7922a4e79cd1b3fe89771477cf0d924ef322c9ed45fad
     or http://127.0.0.1:8891/?token=c4a7922a4e79cd1b3fe89771477cf0d924ef322c9ed45fad
[I 16:21:21.537 LabApp] Build is up to date
[I 16:23:33.759 LabApp] Saving file at /train_loop.py
[W 16:24:51.582 LabApp] Notebook validation.ipynb is not trusted
[I 16:24:52.112 LabApp] Kernel started: 31e7cafa-f3d9-4e7f-a76e-7e29ddaa61ca
[I 16:25:33.792 LabApp] Saving file at /train_loop.py
[I 16:26:52.025 LabApp] Saving file at /validation.ipynb
[W 16:26:52.027 LabApp] Notebook validation.ipynb is not trusted
[I 16:27:34.686 LabApp] Saving file at /train_loop.py
[I 16:27:50.912 LabApp] Saving file at /train_loop.py
[I 17:22:47.301 LabApp] JupyterLab extension loaded from /Users/jacobyeung/anaconda3/lib/python3.8/site-packages/jupyterlab
[I 17:22:47.302 LabApp] JupyterLab application directory is /Users/jacobyeung/anaconda3/share/jupyter/lab
[I 17:22:47.306 LabApp] Serving notebooks from local directory: /Users/jacobyeung/Berkeley/Spring 2021/CS 182/CS-182-Project
[I 17:22:47.306 LabApp] The Jupyter Notebook is running at:
[I 17:22:47.306 LabApp] http://localhost:8890/?token=2a7f0243e0bb595d5b36141ab9a29dc4ff8ca0170dccb91b
[I 17:22:47.306 LabApp]  or http://127.0.0.1:8890/?token=2a7f0243e0bb595d5b36141ab9a29dc4ff8ca0170dccb91b
[I 17:22:47.306 LabApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 17:22:47.326 LabApp] 
    
    To access the notebook, open this file in a browser:
        file:///Users/jacobyeung/Library/Jupyter/runtime/nbserver-1496-open.html
    Or copy and paste one of these URLs:
        http://localhost:8890/?token=2a7f0243e0bb595d5b36141ab9a29dc4ff8ca0170dccb91b
     or http://127.0.0.1:8890/?token=2a7f0243e0bb595d5b36141ab9a29dc4ff8ca0170dccb91b
[I 17:22:50.125 LabApp] Build is up to date
[W 17:22:50.920 LabApp] Notebook validation.ipynb is not trusted
[W 17:22:50.961 LabApp] Notebook timer.ipynb is not trusted
[I 17:22:51.294 LabApp] Kernel started: cc5352e4-4819-455b-85b2-55fd0453da34
[I 17:22:51.308 LabApp] Kernel started: ae9e34dd-bd4a-49a5-9271-17c54cfc6371
[I 17:22:51.321 LabApp] Kernel started: 4e874dfe-9b02-4862-9a98-d846fed58d4c
[I 17:22:52.060 LabApp] Starting buffering for 4e874dfe-9b02-4862-9a98-d846fed58d4c:edd3889c-15d4-45b2-a43f-7218d1c1fbb5
[I 17:23:04.227 LabApp] Saving file at /adv_prune_train_loop.py
[I 17:23:20.327 LabApp] Saving file at /utils.py
[I 17:34:23.496 LabApp] Saving file at /utils.py
[I 20:25:29.926 LabApp] Kernel started: ab007221-01b3-4dcc-9b46-5993dd36e00f
[I 20:46:12.097 LabApp] Starting buffering for 4e874dfe-9b02-4862-9a98-d846fed58d4c:3e8b34b0-5a2b-4997-9bb3-913c3d0f8ab6
[I 20:46:12.526 LabApp] Starting buffering for ab007221-01b3-4dcc-9b46-5993dd36e00f:0bf2e182-65d7-40bd-8902-6db9e87dcddf
[I 20:46:27.558 LabApp] Creating new file in /
[I 20:47:03.710 LabApp] Saving file at /prune_train_loop_tucker.py
[I 22:00:54.526 LabApp] Starting buffering for ab007221-01b3-4dcc-9b46-5993dd36e00f:22312192-2f21-4834-946a-83432ef62adf
[W 22:03:31.166 LabApp] Notebook prune.ipynb is not trusted
[I 22:05:31.823 LabApp] Saving file at /prune.ipynb
[W 22:05:31.824 LabApp] Notebook prune.ipynb is not trusted
[I 00:51:01.164 LabApp] Starting buffering for ae9e34dd-bd4a-49a5-9271-17c54cfc6371:2e52fbf2-3043-4797-9aa4-17901303ab71
[I 00:51:11.179 LabApp] Starting buffering for ab007221-01b3-4dcc-9b46-5993dd36e00f:42a60a8a-1951-44dd-ae96-b25070480095
[I 00:51:45.783 LabApp] Starting buffering for cc5352e4-4819-455b-85b2-55fd0453da34:d81942cd-2d8f-40a8-a19f-54af0580eb26
[I 00:51:45.879 LabApp] Restoring connection for cc5352e4-4819-455b-85b2-55fd0453da34:d81942cd-2d8f-40a8-a19f-54af0580eb26
[I 00:51:45.895 LabApp] Starting buffering for cc5352e4-4819-455b-85b2-55fd0453da34:d81942cd-2d8f-40a8-a19f-54af0580eb26
[I 00:51:47.277 LabApp] Starting buffering for ab007221-01b3-4dcc-9b46-5993dd36e00f:2947d689-eb8d-43de-a21a-b638288f4f31
[I 00:51:47.385 LabApp] Build is up to date
[W 00:51:47.781 LabApp] Notebook validation.ipynb is not trusted
[W 00:51:47.793 LabApp] Notebook timer.ipynb is not trusted
[W 00:51:47.829 LabApp] Notebook prune.ipynb is not trusted
[I 00:51:49.014 LabApp] Discarding 3 buffered messages for cc5352e4-4819-455b-85b2-55fd0453da34:d81942cd-2d8f-40a8-a19f-54af0580eb26
[I 00:51:52.069 LabApp] Starting buffering for ae9e34dd-bd4a-49a5-9271-17c54cfc6371:12e7b93a-f88a-4fe1-bf4e-10879622b3cf
[I 00:52:01.893 LabApp] Starting buffering for ab007221-01b3-4dcc-9b46-5993dd36e00f:55c0fee6-e217-4209-b9bb-5655aa3d7907
[W 00:52:03.268 LabApp] Notebook prune.ipynb is not trusted
[I 00:53:48.305 LabApp] Saving file at /validation.ipynb
[W 00:53:48.305 LabApp] Notebook validation.ipynb is not trusted
[I 01:10:41.392 LabApp] Starting buffering for ab007221-01b3-4dcc-9b46-5993dd36e00f:6af6907a-bbdf-4dc0-a522-5b25fc962ae2
[W 01:10:43.976 LabApp] Notebook prune.ipynb is not trusted
[I 01:11:36.756 LabApp] Saving file at /prune.ipynb
[W 01:11:36.757 LabApp] Notebook prune.ipynb is not trusted
[I 01:11:37.199 LabApp] Starting buffering for ab007221-01b3-4dcc-9b46-5993dd36e00f:d3931760-f703-4798-bcb3-c478a4662a94
[W 01:11:38.423 LabApp] Notebook prune.ipynb is not trusted
[I 01:11:50.799 LabApp] interrupted
Serving notebooks from local directory: /Users/jacobyeung/Berkeley/Spring 2021/CS 182/CS-182-Project
4 active kernels
The Jupyter Notebook is running at:
http://localhost:8890/?token=2a7f0243e0bb595d5b36141ab9a29dc4ff8ca0170dccb91b
 or http://127.0.0.1:8890/?token=2a7f0243e0bb595d5b36141ab9a29dc4ff8ca0170dccb91b
Shutdown this notebook server (y/[n])? [C 01:11:51.904 LabApp] received signal 2, stopping
[I 01:11:51.910 LabApp] Shutting down 4 kernels
[I 01:11:52.324 LabApp] Kernel shutdown: cc5352e4-4819-455b-85b2-55fd0453da34
[I 01:11:52.325 LabApp] Kernel shutdown: ae9e34dd-bd4a-49a5-9271-17c54cfc6371
[I 01:11:52.326 LabApp] Kernel shutdown: 4e874dfe-9b02-4862-9a98-d846fed58d4c
[I 01:11:52.326 LabApp] Kernel shutdown: ab007221-01b3-4dcc-9b46-5993dd36e00f
[I 2021-05-12 01:12:01.980 ServerApp] jupyterlab | extension was successfully linked.
[I 2021-05-12 01:12:02.417 ServerApp] nbclassic | extension was successfully linked.
[I 2021-05-12 01:12:02.477 LabApp] JupyterLab extension loaded from /Users/jacobyeung/anaconda3/envs/ml/lib/python3.7/site-packages/jupyterlab
[I 2021-05-12 01:12:02.478 LabApp] JupyterLab application directory is /Users/jacobyeung/anaconda3/envs/ml/share/jupyter/lab
[I 2021-05-12 01:12:02.482 ServerApp] jupyterlab | extension was successfully loaded.
[I 2021-05-12 01:12:02.490 ServerApp] nbclassic | extension was successfully loaded.
[I 2021-05-12 01:12:02.490 ServerApp] Serving notebooks from local directory: /Users/jacobyeung/Berkeley/Spring 2021/CS 182/CS-182-Project
[I 2021-05-12 01:12:02.490 ServerApp] Jupyter Server 1.2.3 is running at:
[I 2021-05-12 01:12:02.490 ServerApp] http://localhost:8890/lab?token=a8ba0ef091441bd026aecf6e73a3adf30f13ef062eb0dcb1
[I 2021-05-12 01:12:02.490 ServerApp]  or http://127.0.0.1:8890/lab?token=a8ba0ef091441bd026aecf6e73a3adf30f13ef062eb0dcb1
[I 2021-05-12 01:12:02.490 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2021-05-12 01:12:02.513 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///Users/jacobyeung/Library/Jupyter/runtime/jpserver-77653-open.html
    Or copy and paste one of these URLs:
        http://localhost:8890/lab?token=a8ba0ef091441bd026aecf6e73a3adf30f13ef062eb0dcb1
     or http://127.0.0.1:8890/lab?token=a8ba0ef091441bd026aecf6e73a3adf30f13ef062eb0dcb1
[W 2021-05-12 01:12:03.291 ServerApp] Clearing invalid/expired login cookie username-localhost-8890
[W 2021-05-12 01:12:03.291 ServerApp] Couldn't authenticate WebSocket connection
[W 2021-05-12 01:12:03.291 ServerApp] Clearing invalid/expired login cookie username-localhost-8890
[W 2021-05-12 01:12:03.292 ServerApp] Clearing invalid/expired login cookie username-localhost-8890
[W 2021-05-12 01:12:03.303 ServerApp] 403 GET /api/kernels/cc5352e4-4819-455b-85b2-55fd0453da34/channels?session_id=4975dffd-b3f8-483a-b962-556616f81e47&token=2a7f0243e0bb595d5b36141ab9a29dc4ff8ca0170dccb91b (127.0.0.1) 14.39ms referer=None
[I 2021-05-12 01:12:07.911 LabApp] Build is up to date
[W 2021-05-12 01:12:09.171 ServerApp] 404 GET /api/contents/hw1.ipynb?content=0&1620807129163 (127.0.0.1): No such file or directory: hw1.ipynb
[W 2021-05-12 01:12:09.172 ServerApp] No such file or directory: hw1.ipynb
[W 2021-05-12 01:12:09.172 ServerApp] 404 GET /api/contents/hw1.ipynb?content=0&1620807129163 (127.0.0.1) 1.73ms referer=http://localhost:8890/lab
[W 2021-05-12 01:12:09.174 ServerApp] 404 GET /api/contents/hw2.ipynb?content=0&1620807129163 (127.0.0.1): No such file or directory: hw2.ipynb
[W 2021-05-12 01:12:09.174 ServerApp] No such file or directory: hw2.ipynb
[W 2021-05-12 01:12:09.175 ServerApp] 404 GET /api/contents/hw2.ipynb?content=0&1620807129163 (127.0.0.1) 1.67ms referer=http://localhost:8890/lab
[W 2021-05-12 01:12:09.175 ServerApp] 404 GET /api/contents/SDSS_quasar.dat?content=0&1620807129163 (127.0.0.1): No such file or directory: SDSS_quasar.dat
[W 2021-05-12 01:12:09.175 ServerApp] No such file or directory: SDSS_quasar.dat
[W 2021-05-12 01:12:09.175 ServerApp] 404 GET /api/contents/SDSS_quasar.dat?content=0&1620807129163 (127.0.0.1) 1.85ms referer=http://localhost:8890/lab
[W 2021-05-12 01:12:17.169 ServerApp] Notebook prune.ipynb is not trusted
[I 2021-05-12 01:12:18.475 ServerApp] Kernel started: 905a18f9-275d-45cf-86a4-f9ee79991cd5
[W 2021-05-12 01:12:55.745 ServerApp] Notebook prune.ipynb is not trusted
[I 2021-05-12 01:13:05.561 ServerApp] Saving file at /prune.ipynb
[W 2021-05-12 01:13:05.563 ServerApp] Notebook prune.ipynb is not trusted
[I 2021-05-12 01:53:11.336 ServerApp] Saving file at /adv_prune_train_loop2.py
[I 2021-05-12 01:53:12.393 ServerApp] Saving file at /adv_prune_train_loop2.py
